src/platform/OpenThread/GenericThreadStackManagerImpl_OpenThread.cpp

```diff
@@ -980,7 +980,7 @@ CHIP_ERROR GenericThreadStackManagerImpl_OpenThread<ImplClass>::_JoinerStart(voi
 
 #if CHIP_DEVICE_CONFIG_ENABLE_THREAD_SRP_CLIENT
 
-static_assert(OPENTHREAD_API_VERSION >= 80, "SRP Client requires a more recent OpenThread version");
+static_assert(OPENTHREAD_API_VERSION >= 120, "SRP Client requires a more recent OpenThread version");
```

Comment:
Are we able to advance OT version for all vendors? I think if we are about to commit this, we should also update OT submodule and NCS in one PR (or did this before this PR is pushed).

Reply:
Fair point, I updated openthread module. NCS update is not necessary as previous one already contains all we need.